<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RouteTool</class>
 <widget class="QMainWindow" name="RouteTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Route Planner</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenuOphaalpunten">
    <property name="title">
     <string>Ophaalpunten</string>
    </property>
    <addaction name="nieuwOphaalpuntMenuButton"/>
    <addaction name="toonOphaalpuntenMenuButton"/>
   </widget>
   <widget class="QMenu" name="menuMenuAanmeldingen">
    <property name="title">
     <string>Aanmeldingen</string>
    </property>
    <addaction name="nieuweAanmeldingMenuButton"/>
    <addaction name="kiesOphaalpuntenMenuButton"/>
   </widget>
   <widget class="QMenu" name="menuMenuLeveringen">
    <property name="title">
     <string>Leveringen</string>
    </property>
    <addaction name="nieuweLeveringMenuButton"/>
   </widget>
   <widget class="QMenu" name="menuMenuBellijst">
    <property name="title">
     <string>Bellijst</string>
    </property>
    <addaction name="voorspellingMenuButton"/>
    <addaction name="ouderDanEenJaarMenuButton"/>
    <addaction name="wieNogNooitMenuButton"/>
   </widget>
   <addaction name="menuMenuOphaalpunten"/>
   <addaction name="menuMenuAanmeldingen"/>
   <addaction name="menuMenuLeveringen"/>
   <addaction name="menuMenuBellijst"/>
   <addaction name="configurationMenuButton"/>
   <addaction name="actionQuit"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="nieuweAanmeldingMenuButton">
   <property name="text">
    <string>Nieuwe aanmelding</string>
   </property>
  </action>
  <action name="kiesOphaalpuntenMenuButton">
   <property name="text">
    <string>Aanmeldingen</string>
   </property>
  </action>
  <action name="nieuweLeveringMenuButton">
   <property name="text">
    <string>Nieuwe levering</string>
   </property>
  </action>
  <action name="nieuwOphaalpuntMenuButton">
   <property name="text">
    <string>Nieuw ophaalpunt</string>
   </property>
  </action>
  <action name="voorspellingMenuButton">
   <property name="text">
    <string>Voorspelling</string>
   </property>
  </action>
  <action name="ouderDanEenJaarMenuButton">
   <property name="text">
    <string>Ophaalpunten > 1 jaar</string>
   </property>
  </action>
  <action name="wieNogNooitMenuButton">
   <property name="text">
    <string>Punten zonder ophaalhistoriek</string>
   </property>
  </action>
  <action name="configurationMenuButton">
   <property name="text">
    <string>Instellingen</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="toonOphaalpuntenMenuButton">
   <property name="text">
    <string>Ophaalpunten</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
